web/static/js/consent.js

```diff
@@ -3,8 +3,10 @@ var yesConsent = document.querySelectorAll(".yes-consent");
 
 [].forEach.call(noConsent, function(el){
   el.addEventListener("click", function (e) {
-    var index = e.target.id[e.target.id.length - 1];
-    document.getElementById("error-consent-" + index).classList.remove("dn");
+    if(e.target.name[e.target.name.length - 2] === "y") {
```

Comment:
What other scenarios are we catching here other than `e.target.name === "yes"`?
I would find just a `e.target.name` assertion to be easier to read